layouts/html.hbs

```diff
@@ -146,9 +146,31 @@
           } else if (window.isOverlay) {
             window.Overlay.onMessage(message);
           } else if (message.eventType && message.eventType === 'click') {
+            if (ANSWERS) {
+              ANSWERS.components
+                .getActiveComponents('AutoComplete')
+                .forEach(autocompleteComponent => autocompleteComponent.close());
+            }
+          } else if (message.action ===  'update-tab-navigation') {
+            if (!ANSWERS) {
+              return;
+            }
+            function updateTabsLink () {
+              const parentUrlWithoutParams = message.parentUrl.split('?')[0];
+              const navItems = document.getElementsByClassName('js-yxt-navItem');
+              if (navItems) {
+                Array.from(navItems).forEach(navItem => {
+                  const verticalUrl = navItem.pathname.replace(/^\//, '');
+                  const params = navItem.search ? navItem.search : '?';
+                  navItem.href = parentUrlWithoutParams + params + '&verticalUrl=' + verticalUrl;
+                  navItem.target = '_parent';
```

Comment:
I would expect to find the code which defines the navItem hrefs inside the navigation component. If someone is reading the navigation component code, it might be confused to because the navigation component code doesn't include the parentURL, however in practice, it does. They probably wouldn't know to look here to find the code which updates the URLs.

Is it possible to add a function to the Navigation component which would allow you to set the parent URL? That way the Navigation component could take care of updating the URLs.

Reply:
We could probably add it somewhere to the `addComponent` call for the Navigation component  (templates/common-partials/script/navigation.hbs), maybe in an `onMount`? I think adding new config to Navigation might also be a good choice if doing this purely in the theme gets too hacky, though.

Reply:
Having it in the component navigation is tricky when it comes to making sure iframe resizer are initialized so we can get the parent url before updating the tab links. I made changes to SDK and updated this pr to invoke a function to set up the parent url!